core/system/src/mpu/vmpu_armv7m.c

```diff
@@ -360,7 +360,7 @@ void vmpu_acl_stack(uint8_t box_id, uint32_t bss_size, uint32_t stack_size)
     /* allocate context pointer */
     g_context_current_states[box_id].bss = slots_ctx ? box_mem_pos : (uint32_t) NULL;
     /* ensure stack band on top for stack underflow dectection */
-    g_context_current_states[box_id].sp = (box_mem_pos + size - UVISOR_STACK_BAND_SIZE);
+    g_context_current_states[box_id].sp = (box_mem_pos + size) - 8;
```

Comment:
I'm opening an issue for this, since I believe the intention for the stack band and the implementation diverged, especially now that the box stack isn't even used as the main stack anymore.